src/pages/dashboard/charts/_BarChart.js

```diff
@@ -37,21 +40,27 @@ const BarChart = ({
     xScaleDomain.length,
     Math.floor(xScaleDomain.length / xTicks),
   )
+  const textColor = { textColor: colors.text }
   return (
-    <div>
+    <div align={align}>
```

Comment:
What does the `align` attribute do in this case on the `div`?

Reply:
the `align` allows for left / center / right alignment within the surrounding `div`, so that the chart on the left (CDC data) can be right aligned and the right hand chart (tests administered) can be left aligned, so together they are in the center of the page

![image](https://user-images.githubusercontent.com/3722037/79269250-21fbba80-7e6a-11ea-8aed-706ef3a2abdd.png)

Reply:
Woah, maybe I'm missing something but how does it do that??